### DISCUSSION TITLE
spotify connect

### DISCUSSION THREAD

**Discussion by tjengbudi** [rern/rAudio#1718] [Feature Requests]
hello
may i ask about spotify support apps. 
why using spotifyd instead spotify connect? as user, spotify connect will be more easy to use. may this supported with raudio in the feature?

the spotify connect i mean is like this https://github.com/dtcooper/raspotify or use case like moode audio. is it possible to used in raudio?

**Comment by tjengbudi** [comment]
Looks like it's already supported by spotifyd. However it needs to set zeroconf port and install mdns.

Can it support by raudio?

**Comment by tjengbudi** [comment]
i try to force configure the zeroconf of spotify connect
after i config zeroconf port, force spotifyd to run
the spotify desktop can show that there is spotify speaker raudio. however i cannot see it in spotify app android device

after that i try to connect it. and it seems cannot connect
so i try to search why. trying to looks at log

spotify can connect with raudio. however source of spotify is 44.1K rate
i using USB DAC that only support 48k rate. so it cannot playing cause of that.

i see that with alsa i can resampling. but i really have lack of knowledge about this

could you help me ?

**Reply by kan-ibal** [reply]
There must be another reason. The 44.1k rate sound can be played on 48k set soundcard, sometimes sound is scratchy but still.
For resampling see Settings->Player->Sox
